Utest/rimea_tests/test_15/runtest_rimea_15.py

```diff
@@ -117,6 +104,7 @@ def create_dia (peds):
     start_time=time.time()
     test = JPSRunTestDriver(15, argv0=argv[0], testdir=sys.path[0], utestdir=utestdir)
     test.run_test(testfunction=run_rimea_test15)
-    create_dia(500)
     logging.info("%s exits with SUCCESS\nExecution time %.3f seconds." % (argv[0],time.time()-start_time))
     exit(SUCCESS)
+
+create_dia(500)
```

Comment:
This looks wrong. You have 
```python
if __name__ == "__main__":
    ...
    exit(SUCCESS)
```
so that line `create_dia(500)` will only be called if this file is imported instead of executed. Is this intended?

Reply:
No it isn't. I took it back up i the if-statement.